### PR TITLE
Add translations for dynamic strings with l10n

### PR DESCRIPTION
**Pull request by reetchow**
This PR adds l10n into the sample-angular-app to translate things that aren't static strings in the template

### CODE REVIEW

File: webClient/src/app/app.module.ts [hunk 1]
```diff
@@ -11,14 +11,37 @@
...
+import { TranslationModule, L10nConfig, ISOCode, L10nLoader, LOCALE_CONFIG,
+  TRANSLATION_CONFIG, LocaleConfig, TranslationConfig } from 'angular-l10n';
+import { Angular2L10nConfig, Angular2InjectionTokens } from 'pluginlib/inject-resources';
+
+
+const l10nConfig: L10nConfig = {
+  locale: {
+      languages: [
+          { code: 'en', dir: 'ltr' },
+          { code: 'ja', dir: 'ltr' }
```
Comment (1000TurquoisePogs) [inline webClient/src/app/app.module.ts]: This seems like it is missing languages. What is the effect?

File: webClient/src/app/app.component.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
       <zlux-button class="iframe-button shadowed" type="button" (click)="sendAppRequest()"
       i18n="send request to application|Send a request to another application on the desktop@@send-app-request">Send App Request</zlux-button>
       <zlux-button class="iframe-button shadowed" type="button" (click)="getDefaultsFromServer()"
-      i18n="get from server@@get-from-server">Get from Server</zlux-button>
+      i18n="get from server@@plugin-loading-failed-message">Get from Server</zlux-button>
```
Comment (1000TurquoisePogs) [inline webClient/src/app/app.component.html]: Though you replaced the string everywhere so it probably works, Get From Server does not mean plugin loading failed message.

File: webClient/src/app/app.module.ts (hunk 1 above)
Comment (reetchow) [reply webClient/src/app/app.module.ts]: No bearing on the output, will be removed

File: webClient/src/app/app.component.html (hunk 2 above)
Comment (reetchow) [reply webClient/src/app/app.component.html]: fixed

**Pull request merged**